scraper/src/documentation_spider.py

```diff
@@ -41,15 +55,34 @@ def __init__(self, config, algolia_helper, strategy, *args, **kwargs):
         )
 
         DocumentationSpider.rules = [
-            Rule(link_extractor, callback=self.add_records, follow=True),
+            Rule(link_extractor, callback=self.parse_from_start_url, follow=True),
         ]
 
+        # START _init_ part from SitemapSpider
+        # TODO check config attribute
+
+        # In case we don't have a special documentation regex, we assume that start_urls are there to match a documentation part
+        self.documentation_regex = config.documentation_regex if config.documentation_regex is not None else self.start_urls
```

Comment:
Lets add a condition on this to make sure the code is not called when there is no sitemaps